product_docs/docs/pgd/5/durability/commit-scopes.mdx

```diff
@@ -23,11 +23,11 @@ Finally there is the rule which defines what kind of commit scope or combination
 So if a commit scope has a rule that reads:
 
     origin_node_group := 'example_bdr_group',
-    rule := 'ANY 2 (example_bdr_group) GROUP COMMIT',
+    rule := 'MAJORITY (example_bdr_group) GROUP COMMIT',
```

Comment:
@mwanner help me understand is this breaking change?

Reply:
No, this is us changing docs to guide users to options we want them to use rather than options that exist only to solve corner cases.